pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/UserDao.java

```diff
@@ -97,6 +97,11 @@ private User createUserByClientIdOrAuth0Ids(boolean byClientId, Long clientId,
     @RegisterConstructorMapper(User.class)
     Optional<User> findUserByAuth0UserId(@Bind("auth0UserId") String auth0UserId, @Bind("auth0TenantId") long auth0TenantId);
 
+    @UseStringTemplateSqlLocator
+    @SqlQuery("queryUserByGuidOrAltPid")
+    @RegisterConstructorMapper(User.class)
+    Optional<User> findUserByGuidOrAltPid(@Bind("guidOrAltPid") String userGuidOrAltPid);
```

Comment:
Did you check out existing: JdbiUser.findByLegacyAltPidIfNotFoundByUserGuid(String userGuidOrLegacyAltPid)

Reply:
I did more refactor and removed `JdbiUser.findByLegacyAltPidIfNotFoundByUserGuid()` in favor of `UserDao`.

Reply:
noticed that later. looks good